packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -216,7 +216,7 @@ const ReflectionCard = (props: Props) => {
     phases,
     isSpotlightSource
   )
-  const userSelect = readOnly ? (phaseType === 'discuss' ? 'text' : 'none') : undefined
+  const userSelect = readOnly ? (phaseType === 'discuss' || 'vote' ? 'text' : 'none') : undefined
```

Comment:
-1 I didn't quite get the logic here, when the `phaseType` is `vote`, the result of `'vote' === 'discuss'` is FALSE and then the default value is taken as `'vote'`? This shouldn't be what we want.

Reply:
Thank you @JimmyLv for clarifying the difference between the two:
My code made reflection cards selectable in every phase.